test/assets/preset-wind-targets.ts

```diff
@@ -394,7 +394,6 @@ export const presetWindTargets: string[] = [
 
   // variants experimental
   '@hover-text-red',
-  '@hover:[[open]_&]:text-blue',
```

Comment:
We should add a separate test for this in `preset-wind.test.ts`.

Reply:
It has been moved to
https://github.com/unocss/unocss/blob/350fd1a38b0ae8c2be3de3f23a63db31e92235df/test/assets/preset-arbitrary-target.ts#L16-L17